infra/k8s/README.md

```diff
@@ -79,13 +79,19 @@ kubectl apply -f ./flannel.yml
 kubectl apply -f ./genie-plugin.yaml
 ```
 
-9. Install Weave
+9. Install Dummy daemonset - we need a container on every worker node so that interface `cni0` is created, and Weave's initContainer can add a route to the Services CIDR
```

Comment:
Let's expand the explanation here. I'm still not sure why this is necessary.

Reply:
If you look at the script from above, you will see that we use `cni0` interface to detect what gateway to use for the Services CIDR (located at 100.64.0.0/16). `cni0` does not exist before you've allocated at least one pod on a given host, so this is what this daemonset is doing.

We could remove the daemonset after that, because it is no longer needed.

Reply:
Ah, now I see. Could we just use the sidecar?

Reply:
Oh, we can't because that uses host routing. Makes sense.